python/ikarus/_ikarus.cc

```diff
@@ -68,6 +68,19 @@ PYBIND11_MODULE(_ikarus, m) {
   fePar.value("loadfactor", FEParameter::loadfactor);
   fePar.value("time", FEParameter::time);
 
+  py::enum_<ResultType> resType(m, "ResultType");
```

Comment:
Can you write here a loop as given in https://github.com/ikarus-project/ikarus/blob/35d611474a4dd295f32277ce7f2f9b94dc079590/ikarus/utils/observer/observer.hh#L51 using the toString function from here https://github.com/ikarus-project/ikarus/blob/35d611474a4dd295f32277ce7f2f9b94dc079590/ikarus/utils/makeenum.hh#L27 ? can you also add this for the other enums otherwise this has to be refactored all the time if we add something. Sorry to bother you here within this PR.

Reply:
Good idea, i might want to do this in a second PR, i think the python bindings need a bit more of housekeeping anyway

Reply:
I would rather keep the enums to have a unified interface between python and cpp